metadata.rb

```diff
@@ -3,7 +3,7 @@
 maintainer_email  'help@sous-chefs.org'
 license           'Apache-2.0'
 description       'Installs and manages Logical Volume Manager'
-version           '6.1.0'
+version           '6.1.1'
```

Comment:
Can you revert this? The pipeline will auto increment this based on a PR label.

Reply:
Will do